5calls/app/src/main/java/org/a5calls/android/a5calls/net/FiveCallsApi.java

```diff
@@ -231,4 +242,22 @@ private void onRequestError(VolleyError error) {
             Log.d("Error", error.getMessage());
         }
     }
+
+    private void filterSkipOutcomes(List<Issue> issues) {
```

Comment:
This is pretty unrelated to the API. Please refactor into a static method in the Outcome class instead.

Reply:
Done.